src/tsolvers/THandler.cc

```diff
@@ -120,14 +119,15 @@ void THandler::getConflict (
     // where li is the literal corresponding with ei with polarity !pi
     vec<PtAsgn> explanation;
     {
-        int i;
-        for (i = 0; i < getSolverHandler().tsolvers.size(); i++) {
-            if (getSolverHandler().tsolvers[i] != nullptr && getSolverHandler().tsolvers[i]->hasExplanation()) {
-                getSolverHandler().tsolvers[i]->getConflict(explanation);
-                break;
+        bool found;
+        for (auto solver : getSolverHandler().solverSchedule) {
+            if (solver->hasExplanation()) {
+                solver->getConflict(explanation);
+                found = true;
```

Comment:
I think we should keep the `break` here, no?

Reply:
True, no need to remove it.  I also seem to have forgotten to initialize `found`, fixed in the commit below.